ui/page/livestreamOverlay/view.jsx

```diff
@@ -0,0 +1,301 @@
+  fetchCreatorSettings: (channelId: string) => void,
+};
+
+export default function SettingsOverlayPage(props: Props) {
+  const { activeChannelClaim, settingsByChannelId, fetchCreatorSettings } = props;
+
+  const [lastUpdated] = React.useState(1);
+
+  // **************************************************************************
+  // **************************************************************************
+
+  // Re-sync list on first idle time; mainly to correct any invalid settings.
+  React.useEffect(() => {
+    if (lastUpdated && activeChannelClaim) {
+      const timer = setTimeout(() => {
+        fetchCreatorSettings(activeChannelClaim.claim_id);
+      }, DEBOUNCE_REFRESH_MS);
+      return () => clearTimeout(timer);
+    }
+  }, [lastUpdated, activeChannelClaim, fetchCreatorSettings]);
+
+  // **************************************************************************
+  // **************************************************************************
+
+  const isBusy =
+    !activeChannelClaim || !settingsByChannelId || settingsByChannelId[activeChannelClaim.claim_id] === undefined;
+  const isDisabled =
+    activeChannelClaim && settingsByChannelId && settingsByChannelId[activeChannelClaim.claim_id] === null;
+
+  // **************************************************************************
+  // **************************************************************************
+  return (
+    <Page
+      noFooter
+      backout={{ title: __('Overlay settings and instructions'), backLabel: __('Back') }}
+      className="card-stack"
+    >
+      <div className="card-stack">
+        <ChannelSelector hideAnon />
+
+        {isBusy && (
+          <div className="main--empty">
+            <Spinner />
+          </div>
+        )}
+
+        {isDisabled && (
+          <Card
+            title={__('Settings unavailable for this channel')}
+            subtitle={__("This channel isn't staking enough LBRY Credits to enable Overlay settings and instructions.")}
+          />
+        )}
+
+        {!isBusy && !isDisabled && (
+          <>
+            <div className="card__title-section">
+              <h2 className="card__title">{__('Chat Overlay')}</h2>
+            </div>
+            <Card
+              body={
+                <>
+                  <SettingsRow title={__('Chat')}>
+                    <FormField
+                      name="chat_overlay"
+                      type="checkbox"
+                      checked={__('true')} /* Default value */
+                      onChange={(e) => { }}
+                      />
+                  </SettingsRow>
+                  <p className="help--inline">
+                    <em>{__(HELP.CHAT)}</em>
+                  </p>
+
+                  <SettingsRow title={__('Position')}>
+                    <FormField
+                      name="chat_overlay_position"
+                      type="select"
+                      value={__('Left')} /* Default value */
```

Comment:
This combo box is always stuck at `Left`
(I think you mentioned something about this, so seems like a known WIP)